test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,391 @@ describe('getDefaultAliasIndex', () => {
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workB.toJSON().bbid,
+			typeId: 1
+		};
+
+		// Relate Relationship Data with Relationship Type
+		relationshipDataA.typeId = relationshipTypeData.id;
+		relationshipDataA.id = random.number();
+
+		relationshipDataB.typeId = relationshipTypeData.id;
+		relationshipDataB.id = random.number();
+
+		// Create Relationships for Entities
+		const relationshipA = await new Relationship(relationshipDataA)
+			.save(null, {method: 'insert'});
+
+		const relationshipB = await new Relationship(relationshipDataB)
+			.save(null, {method: 'insert'});
+
+		// Create Relationship set ids for entities
+		const authorRelationshipSetId = random.number();
+		const workARelationshipSetId = random.number();
+		const workBRelationshipSetId = random.number();
+
+		// Create Relationship sets
+		let authorRelationshipSet = await new RelationshipSet({id: authorRelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationshipA, relationshipB]).then(() => model)
+			);
+		authorRelationshipSet = await authorRelationshipSet.refresh({withRelated: 'relationships'});
+
+		let workARelationshipSet = await new RelationshipSet({id: workARelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationshipA]).then(() => model)
+			);
+		workARelationshipSet = await workARelationshipSet.refresh({withRelated: 'relationships'});
+
+		let workBRelationshipSet = await new RelationshipSet({id: workBRelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationshipB]).then(() => model)
+			);
+		workBRelationshipSet = await workBRelationshipSet.refresh({withRelated: 'relationships'});
+
+		// Set the Entity Attributes as Required
+		author.set('revisionId', revisionId);
+		author.set('relationshipSetId', authorRelationshipSet.get('id'));
+		workA.set('revisionId', revisionId);
+		workA.set('relationshipSetId', workARelationshipSet.get('id'));
+		workB.set('revisionId', revisionId);
+		workB.set('relationshipSetId', workBRelationshipSet.get('id'));
+
+		// Save the Entities and Update the database
+		await author.save(null, {method: 'update'});
+		await workA.save(null, {method: 'update'});
+		await workB.save(null, {method: 'update'});
+
+		// Get updated Entity
+		workA = await new Work({bbid: workA.get('bbid')}).fetch({withRelated: 'relationshipSet.relationships'});
+
+		// Get list of Affected Entities
+		const affectedEntities = await deleteRelationships(orm, null, workA.toJSON());
+
+		const affectedEntityRelationshipSet = await new RelationshipSet({
+			id: affectedEntities[0].get('relationshipSetId')
+		}).fetch({withRelated: 'relationships'});
+
+		expect(affectedEntities[0].get('bbid')).to.equal(author.get('bbid'));
+		expect(affectedEntityRelationshipSet.toJSON().relationships[0].targetBbid).to.be.equal(workB.get('bbid'));
```

Comment:
Here you can serialize only the relationships: `affectedEntityRelationshipSet.related('relationships').toJSON()[0].targetBbid`